src/mercury_engine_data_structures/formats/bmsas.py

```diff
@@ -140,17 +172,72 @@ def _emitbuild(self, code: construct.CodeGen):
     )),
 )
 
-BMSAS = Struct(
+Action = Struct(
+    action_name=StrIdSR,
+    bcskla=StrIdSR,
+)
+
+AnimationSR = Struct(
```

Comment:
Is there _anything_ shared between Dread and SR in this file? Thinking could just split into `bmsas_sr` and `bmsas_dread`, with maybe a `bmsas_common`.

Reply:
they're structurally similar in key ways and I prefer having those similarities made more apparent by having the different versions next to each other

Reply:
Different files you can have both side by side, but either works for me